alpaca/entities.go

```diff
@@ -293,6 +294,7 @@ const (
 type TradeUpdate struct {
 	At          time.Time        `json:"at"`
 	Event       string           `json:"event"`
+	EventID     ulid.ULID        `json:"event_id"`
```

Comment:
This is more like a question: what's the benefit of using this type over string from the client's perspective? I don't think it would ever be used to anything other than copy/pasting to sinceID/untilID queries.

Reply:
We're using strings (instead of UUIDs) for all the other ID types in the codebase.

Reply:
Right, given that we support setting the since/until time, it won't give us much. The end user can still parse the ulid if needed. Let me make the necessary chanages.

Reply:
Fixed, please re-check the PR.